### PR TITLE
 feat(users): Adds initial implementation of user related schemas

### PR DESCRIPTION
**Pull request by lupomontero**
Includes the following new schemas:

* `OrganizationSchema`
* `GraduateProfileEndorsementSchema`
* `GraduateProfileLifeSkillSchema`
* `GraduateProfileProjectSchema`
* `GraduateProfileSchema`

See #1

### CODE REVIEW

**Comment by lupomontero** [comment]
Since the data in `StudentProfileSchema`, `StudentProfileEndorsementSchema`, `StudentProfileProjectSchema` and `StudentProfileLifeSkillSchema` is used exclusively as part of the _job placement_ process (data is managed in LMS but displayed in job placement app), I am seriously considering renaming them as follows:

* `JobPlacementProfileSchema`
* `JobPlacementProfileEndorsementSchema`
* `JobPlacementProfileProjectSchema`
* `JobPlacementProfileLifeSkillSchema`

Thoughs?

cc/ @bouli @gmoura @Xpktro @MaiaRojas

File: src/StudentProfileEndorsementSchema.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+//     "endorsedBy": "ObjectId",
+//     "organization": "ObjectId",
+//     "text": "Blah blah blah..."
+//   }
+// ]
+//
+
+module.exports = (conn) => {
+  const StudentProfileEndorsementSchema = new conn.Schema({
+    student: {
```
Comment (lupomontero) [inline src/StudentProfileEndorsementSchema.js]: If we change the schema name to `JobPlacementProfileEndorsementSchema`, would it make sense to rename this _field_ to `user` or even `endorsee`... :thinking: 

cc/ @gmoura

File: src/StudentProfileLifeSkillSchema.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+//     "endorsedBy": "ObjectId",
+//     "organization": "ObjectId",
+//     "skill": "selfLearning"
+//   }
+// ]
+//
+
+module.exports = (conn) => {
+  const StudentProfileLifeSkillSchema = new conn.Schema({
+    student: {
```
Comment (lupomontero) [inline src/StudentProfileLifeSkillSchema.js]: See https://github.com/Laboratoria/schemas/pull/4/files#r269727559

File: src/StudentProfileProjectSchema.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+//     "image": "http://foo.bar/baz.png",
+//     "where": "Laboratoria",
+//     "date": "2018-11"
+//   }
+// ]
+//
+
+module.exports = (conn) => {
+  const StudentProfileProjectSchema = new conn.Schema({
+    student: {
```
Comment (lupomontero) [inline src/StudentProfileProjectSchema.js]: If we change the schema name to `JobPlacementProfileEndorsementSchema`, would it make sense to rename this _field_ to simply `user`?

cc/ @gmoura

File: src/StudentProfileSchema.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+//   `aboutMe.highlights.biggestAchievements` in the `users` collection.
+// * `interests`: Previously known as `aboutMe.highlights.areaOfInterest` in the
+//   `users` collection.
+// * `employmentProfile`: Previously known as `recommendedAs` or `recomendedAs`
+//    in the `users` collection.
+//
+
+module.exports = (conn) => {
+  const StudentProfileSchema = new conn.Schema({
+    student: {
```
Comment (lupomontero) [inline src/StudentProfileSchema.js]: See https://github.com/Laboratoria/schemas/pull/4/files#r269728191

File: src/StudentProfileEndorsementSchema.js (hunk 1 above)
Comment (arku) [reply src/StudentProfileEndorsementSchema.js]: @lupomontero At this lifecycle stage, I am not sure if sticking with the term `student` makes sense. What do you think of using the term `graduate` instead? With this change, we don't have to use `JobPlacementProfileEndorsementSchema` but just `GraduateProfileEndorsementSchema`.

I might be missing something, feel free to correct me if I am wrong.

/cc @gmoura

Comment (lupomontero) [reply src/StudentProfileEndorsementSchema.js]: `StudentProfile` has now been renamed to `GraduateProfile` :wink:

**Comment by lupomontero** [comment]
Después de la votación en Slack nos quedamos con `GraduateProfile` :tada:

image

**Review by gmoura — APPROVED** [review]

**Review by MaiaRojas — APPROVED** [review]

**Review by bouli — APPROVED** [review]
Vamooooooooooooooooooooooos!!!

**Pull request merged**